axelrod/result_set.py

```diff
@@ -45,6 +47,8 @@ def __init__(self, players, interactions, progress_bar=True):
             progress_bar : bool
                 Whether or not to create a progress bar which will be updated
         """
+        if game is not None:
+            self.game = game
```

Comment:
No problemo :)

On Mon, Jul 18, 2016 at 11:40 AM Owen Campbell <notifications@github.com>
wrote:

> In axelrod/result_set.py
> <https://github.com/Axelrod-Python/Axelrod/pull/662#discussion_r71127725>:
>
> > @@ -45,6 +47,8 @@ def __init__(self, players, interactions, progress_bar=True):
> >              progress_bar : bool
> >                  Whether or not to create a progress bar which will be updated
> >          """
> > +        if game is not None:
> > +            self.game = game
>
> Which is what I should have suggested in the first place, but was too lazy
> to type out in full. sorry.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/Axelrod-Python/Axelrod/pull/662/files/f8fb35d1f9fcfc563b1d453dd9128a3078d02ec3#r71127725>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ACCGWqLw8lgwtJDxZTtmzUmLXiNJ95DGks5qW1gSgaJpZM4JOEOa>
> .
>